### PR TITLE
1726 fixed img, linter errors, and duplicate heading

### PR DESCRIPTION
**Pull request by tristandenyer**
Fixes Issue #1726 

— updated broken img and img link
— fixed linter errors
— fixed duplicate heading
— linter / prettyfier adjusted tables and unordered lists

### CODE REVIEW

File: v6/postman/collections/data_formats.md [hunk 1]
```diff
@@ -97,35 +97,31 @@ A Swagger API definition usually lives as a single file, so we only support impo
...
-   3.  Click on file and upload the Swagger file.
+3. Click on file and upload the Swagger file.
 
-  You’re done! Postman will detect all the Swagger definitions and convert them internally to Postman and then show you an import success message.
+You're done! Postman will detect all the Swagger definitions and convert them internally to Postman and then show you an import success message.
 
 [![confirmation message](https://s3.amazonaws.com/postman-static-getpostman-com/postman-docs/WS-data-format-raml-2a.png)](https://s3.amazonaws.com/postman-static-getpostman-com/postman-docs/WS-data-format-raml-2a.png)
 
-#### Examples
+#### Example Swagger files
```
Comment (kaustavdm) [inline v6/postman/collections/data_formats.md]: We should call these `OpenAPI` files.

Comment (tristandenyer) [reply v6/postman/collections/data_formats.md]: Fixed! @kaustavdm

**Comment by tristandenyer** [comment]
@kaustavdm should every instance of "Swagger" be changed to "OpenAPI"?

**Comment by kaustavdm** [comment]
@tristandenyer Yes. We should be talking about the OpenAPI specification. Swagger is the toolset around OpenAPI.

File: v6/postman/collections/data_formats.md (hunk 1 above)
Comment (tristandenyer) [reply v6/postman/collections/data_formats.md]: Updated all mentions of Swagger to now be OpenAPI. @kaustavdm please review that it still reads correctly.

**Comment by tristandenyer** [comment]
> @tristandenyer Yes. We should be talking about the OpenAPI specification. Swagger is the toolset around OpenAPI.

Updated all mentions of Swagger to now be OpenAPI. @kaustavdm please review that it still reads correctly.

**Pull request merged**